### DISCUSSION TITLE
Navigation shows blank page with current year from Footer instead of the new page

### DISCUSSION THREAD

**Discussion by macellinoss275** [livewire/livewire#7700] [1. Help]
I'm using Vuexy (Laravel version) template with Livewire 3. I've been able to successfully integrated it. Then I tried to make 2 pages, Blogs Index and Blogs Create. Blogs Index is for list of Blogs, and Blogs Create is the page form for creating a new blog. The thing is, when I clicked on "Create Blogs" button, instead of navigating to the new Blogs Create page, it only displayed "2024", which is the current year in my Footer. Why this happened?

This is a 3 second video showing exactly what I meant:
https://github.com/livewire/livewire/assets/138418079/ba5f44ba-0ea2-4f39-b50a-bc3490168f4c

This is my Blogs Index Livewire PHP code that use layoutMaster.blade.php as the layout:
```
<?php

namespace App\Livewire\Blogs;

use Livewire\Attributes\Layout;
use Livewire\Component;

class Index extends Component
{
    #[Layout('layouts.layoutMaster')]
    public function render()
    {
        return view('content.blogs.index');
    }
}
```

This is my Blogs Create Livewire PHP code:
```
<?php

namespace App\Livewire\Blogs;

use Livewire\Attributes\Layout;
use Livewire\Component;

class Create extends Component
{
    #[Layout('layouts.layoutMaster')]
    public function render()
    {
        return view('content.blogs.create');
    }
}
```

Which according to Vuexy's documentation, this should be correct:
![image](https://github.com/livewire/livewire/assets/138418079/7dd6a4f9-d089-4ae5-9561-08169aaad56e)

This is my content\blogs\index.blade.php code:
```
@extends('layouts/layoutMaster')

@section('title', 'Blogs - Index')

@section('content')
  <div class="row">
    <a href="/blogs/create" wire:navigate>Create Blogs</a>
  </div>
@endsection
```

And finally this is my content\blogs\create.blade.php code:
```
@extends('layouts/layoutMaster')

@section('title', 'Blogs - Create')

@section('content')
  <div class="row">
    <a href="/blogs" wire:navigate>Back to Index</a>
  </div>
@endsection
```

Can someone please help me about this problem? I don't understand why Livewire navigate only display the current year from Footer instead of the whole page.

**Comment by mundua** [comment]
Hello, Having same issue, were you able to resolve this?

**Comment by masihgh** [comment]
i have this problem also in livewire 3 and Frest Theme :|

**Comment by masihgh** [comment]
Consult the footer section, print year using JavaScript. I hope it works by removing the script tag. footer.blade.php

It worked for me.
